### DISCUSSION TITLE
Create jobs that backup to Azure blob

### DISCUSSION THREAD

**Discussion by nishtechgithub** [olahallengren/sql-server-maintenance-solution#318] [Questions & Answers]
Are there options or parameters that can be set in MaintenanceSolution.sql that will cause the jobs to be created to backup to Azure blob storage?

**Comment by olahallengren** [comment]
MaintenanceSolution.sql does not have options to create jobs to backup to Azure Blob Storage. You will have to create the jobs or modify the existing jobs.

The jobs that comes with MaintenanceSolution.sql are just to make it easy to get started. They don't support all the options in the stored procedures.
